docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -25,11 +25,18 @@ To run Hive on Hadoop MapReduce, please also follow the instructions in
 [running MapReduce on Alluxio](Running-Hadoop-MapReduce-on-Alluxio.html) to make sure Hadoop
 MapReduce can run with Alluxio.
 
-Hive users can either create
+## Configure Hive
+
+First of all, set `HIVE_AUX_JARS_PATH` either in shell or `conf/hive-env.sh`:
+
+```bash
+export HIVE_AUX_JARS_PATH={{site.ALLUXIO_CLIENT_JAR_PATH}}:${HIVE_AUX_JARS_PATH}
+```
+Alluxio can be used as storage for both
 [external tables](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-ExternalTables)
-that point to specified locations in Alluxio while keeping the storage of other tables unchanged,
-or use Alluxio as the default filesystem to operate on. In the following, we will introduce these
-two approaches to use Hive with Alluxio.
+and internal tables. These tables can be new tables that are being created or existing tables that
+are stored in HDFS. Alluxio can also be used as the default file system for Hive. In the following
+sections, we will describe how to use Hive with Alluxio for these use cases. 
 
 ## Create External Table Located in Alluxio
```

Comment:
-> "Create External Table from Files Located in Alluxio" to be consistent with other section titles?